subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt

```diff
@@ -1,99 +1,100 @@
-import org.junit.jupiter.api.io.TempDir
-import java.io.File
-
-internal class TestExecutorIntegrationTest {
-
-    private val loggerFactory = StubLoggerFactory
-
-    @Test
-    fun `executor - does not stuck - when no requests passed`(@TempDir tempDir: File) {
-        val executor = createTestExecutor(loggerFactory = loggerFactory)
-
-        executor.execute(
-            application = null,
-            testApplication = File(""),
-            testsToRun = listOf(),
-            executionParameters = ExecutionParameters.createStubInstance(),
-            output = tempDir
-        )
-
-        // no assertion needed: just stuck in runBlocking forever before fix
-    }
-
-    @Test
-    fun `executor - does not stuck - when deployment failed with invalid image reference`(@TempDir tempDir: File) {
-        val executor = createTestExecutor(loggerFactory = loggerFactory)
-
-        executor.execute(
-            application = null,
-            testApplication = File(""),
-            testsToRun = listOf(
-                TestWithTarget(
-                    test = TestStaticDataPackage.createStubInstance(),
-                    target = TargetConfiguration.Data.createStubInstance(
-                        reservation = Reservation.StaticReservation(
-                            device = Device.CloudEmulator.createStubInstance(image = "invalid/image/reference"),
-                            count = 1,
-                            quota = QuotaConfiguration.Data(
-                                retryCount = 0,
-                                minimumSuccessCount = 1,
-                                minimumFailedCount = 0
-                            )
-                        )
-                    )
-                )
-            ),
-            executionParameters = ExecutionParameters.createStubInstance(),
-            output = tempDir
-        )
-
-        // no assertion needed: just stuck in runBlocking forever before fix
-    }
-
-    private fun createTestExecutor(
-        loggerFactory: LoggerFactory,
-        configurationName: String = "",
-        buildId: String = "integration-test-build-id",
-        statsDConfig: StatsDConfig = StatsDConfig.Disabled,
-        adb: Adb = Adb()
-    ): TestExecutor = TestExecutor.Impl(
-        devicesProvider = KubernetesDevicesProvider(
-            client = KubernetesReservationClient.createStubInstance(
-                loggerFactory = loggerFactory,
-                adb = adb
-            ),
-            adbDevicesManager = AdbDevicesManager(
-                loggerFactory = loggerFactory,
-                adb = adb
-            ),
-            loggerFactory = loggerFactory,
-            adb = adb
-        ),
-        testReporter = StubTestReporter(),
-        buildId = buildId,
-        configurationName = configurationName,
-        loggerFactory = loggerFactory,
-        statsDConfig = statsDConfig
-    )
-}
+//import com.avito.android.stats.StatsDConfig
```

Comment:
@eugene-krivobokov @dsvoronin 
Discussion about that https://github.com/avito-tech/avito-android/discussions/738